### PR TITLE
Project 1 Part 1

### PR DESCRIPTION
**Pull request by sbl-plugins**
Added 10 nouns and verbs and executed the nouns and verbs in pseudo code.

Slack username: soundbylaura

### CODE REVIEW

File: main.cpp [hunk 1]
```diff
@@ -51,107 +51,115 @@ arm.flex();                             // 3)
...
-
-
-
+//  Noun: programmer
+//  action 1: the programmer writes code
+    programmer.writeCode();
+//  action 2: the programmer fixes their code
+    programmer.fixBug();
+//  action 3: the programmer retires
+    programmer.retire();
```
Comment (matkatmusic) [inline main.cpp]: As explained in the instructions:
Here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
REPLACE the highlighted noun/action as follows:
- [x] come up with a new noun and 3 actions
- [x] pick a noun that needs `camelCase` for the name
- [x] pick actions that need `camelCase` for their name.

Do not create an 11th noun.

Once you have completed the above-requested changes, please do the following:
- click each checkbox to mark each requested change as completed.
- click the `[Resolve Conversation]` button.
- Re-request a review of your pull request. It's on the right side of the Pull request page where you originally requested my review.
- Send me the link to your pull request in a new message.

File: main.cpp [hunk 2]
```diff
@@ -51,107 +51,115 @@ arm.flex();                             // 3)
...
+    riceKrispies.pop();
+
+//  9) 
+//  Noun: 3d printer
+//  action 1: the 3d printer moves along the X axis
+    3dPrinter.moveX();
+//  action 2: the 3d printer auto-levels
+    3dPrinter.autoLevel();
+//  action 3: the 3d printer shuts off
+    3dPrinter.off();
```
Comment (matkatmusic) [inline main.cpp]: Numbers can't be the first character in the name of a variable
- [x] revise

File: main.cpp [hunk 3]
```diff
@@ -51,107 +51,115 @@ arm.flex();                             // 3)
...
-//  action 2:
-
-//  action 3:
+//  Noun: slot machine
+//  action 1: the slot machine reels spin
+    slotMachineReels.spin();
+//  action 2: the slot machine reels stop
+    slotMachineReels.stop();
+//  action 3: the slot machine bell rings
+    slotMachineBell.ring();
```
Comment (matkatmusic) [inline main.cpp]: The name of the object in the pseudo-code does not match the name of the noun
- [x] revise

File: main.cpp [hunk 4]
```diff
@@ -51,107 +51,115 @@ arm.flex();                             // 3)
...
-
-//  action 3:
+//  1) 
+//  Noun: cat
+//  action 1: the cat purrs
+    cat.purr()
+//  action 2: the cat stretches
+    cat.stretch();
+//  action 3: the cat chases a ball
+    cat.chaseBall
```
Comment (matkatmusic) [inline main.cpp]: where's the `();`?
- [x] revise

**Review by matkatmusic — CHANGES_REQUESTED** [review]

File: main.cpp (hunk 1 above)
Comment (sbl-plugins) [reply main.cpp]: Done.

File: main.cpp (hunk 2 above)
Comment (sbl-plugins) [reply main.cpp]: Fixed.

File: main.cpp (hunk 3 above)
Comment (sbl-plugins) [reply main.cpp]: Fixed.

File: main.cpp (hunk 4 above)
Comment (sbl-plugins) [reply main.cpp]: Fixed.

**Review by matkatmusic — APPROVED** [review]

**Pull request merged**